src/zope/cachedescriptors/property.txt

```diff
@@ -237,3 +276,20 @@ invalidation:
 
     >>> '%.2f' % point.radius
     '2.24'
+
+Documentation is preserved if the attribute is accessed through
+the class. This allows Sphinx to extract the documentation.
+
+    >>> class Point:
+    ...
+    ...     def __init__(self, x, y):
+    ...         self.x, self.y = x, y
+    ...
+    ...     @property.cachedIn('_radius_attribute')
+    ...     def radius(self):
+    ...         '''The length of the line between self.x and self.y'''
+    ...         print('computing radius')
+    ...         return math.sqrt(self.x**2 + self.y**2)
+
+    >>> print(Point.radius.__doc__)
+    The length of the line between self.x and self.y
```

Comment:
No `__name__` check here?

I'm confused because previous examples imply that both `__doc__` and `__name__` are needed for Sphinx.  The changelog points out that `cachedIn` doesn't preserve everything, but doesn't say _why_.